readme.md

```diff
@@ -91,6 +91,12 @@ BunnyCDNRegion::SINGAPORE = 'sg';
 
 # Oceania
 BunnyCDNRegion::SYDNEY = 'syd';
+
+# Africa
+BunnyCDNRegion::JOHANNESBURG = 'JH';
+
+# South America
+BunnyCDNRegion::BRAZIL = 'BR';
```

Comment:
they should be lowercase, shouldn't they?

Reply:
good catch!